lib/block_view.go

```diff
@@ -1143,7 +1149,7 @@ func (bav *UtxoView) _verifySignature(txn *MsgDeSoTxn, blockHeight uint32) (_der
 	// If derived key is present in ExtraData, we check if transaction was signed by the derived key.
 	if derivedPk == nil {
 		// Verify that the transaction is signed by the specified key.
-		if txn.Signature.Verify(txHash[:], ownerPk) {
+		if txn.Signature.Sign.Verify(txHash[:], ownerPk) {
```

Comment:
instead of updating all the usages of `txn.Signature.Verify` to `txn.Signature.Sign.Verify`, let's create a function on `DeSoSignature` called `Verify`

```
func (desoSignature DeSoSignature) Verify(hash []byte, pubKey *PublicKey) bool {
	return desoSignature.Sign.Verify(hash, pubKey)
}

Reply:
I like this suggestion a lot. Dons